transport/http/src/main/java/org/kaazing/gateway/transport/http/HttpAcceptor.java

```diff
@@ -268,6 +276,8 @@ void bindInternal(final ResourceAddress address, IoHandler handler,
         //
         final ResourceAddress transportAddress = address.getTransport();
 
+        handshakeTimeoutByTransportAddress.put(transportAddress, address.getOption(HANDSHAKE_TIMEOUT).longValue());
```

Comment:
Because resources addresses at HTTP might bind on the same transport address, the conflict resolution for when HTTP handshake timeouts are set may have to be done here...